diff -aurN parted-3.1.orig/libparted/fs/Makefile.am parted-3.1/libparted/fs/Makefile.am
--- parted-3.1.orig/libparted/fs/Makefile.am	2012-11-05 21:28:55.297682374 +0100
+++ parted-3.1/libparted/fs/Makefile.am	2012-11-05 22:33:38.069117116 +0100
@@ -114,6 +114,8 @@
   r/hfs/reloc_plus.c		\
   r/hfs/reloc_plus.h
 
+libparted_fs_resize_la_LIBADD = $(UUID_LIBS)
+
 INCLUDES = \
   -I$(top_srcdir)/libparted/labels	\
   $(partedincludedir)			\
diff -aurN parted-3.1.orig/libparted/Makefile.am parted-3.1/libparted/Makefile.am
--- parted-3.1.orig/libparted/Makefile.am	2012-11-05 21:28:55.297682374 +0100
+++ parted-3.1/libparted/Makefile.am	2012-11-05 22:33:53.860686326 +0100
@@ -58,7 +58,8 @@
   $(DM_LIBS)		\
   $(SELINUX_LIBS)	\
   $(LIB_BLKID)		\
-  $(INTLLIBS)
+  $(INTLLIBS)		\
+  $(UUID_LIBS)
 
 EXTRA_DIST	      = mbr.s
 
